### PR TITLE
Run build once every 24h

### PR DESCRIPTION
**Pull request by praveenkumarct**
Schedule CI build to run on end of every day.

### CODE REVIEW

**Review by butenkor — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,7 +3,10 @@
 
 name: CI
 
-on: [push]
+on:
```
Comment (ahmetoz) [inline .github/workflows/ci.yml:6]: Note that the build would not run for every push, @butenkor @praveenkumarct is this what you wanted?

Comment (praveenkumarct) [reply .github/workflows/ci.yml:6]: I am still working on this, Build was not working for the schedule. So I tried this. 
Will ask for the review, Once i am done with the changes.

Comment (butenkor) [reply .github/workflows/ci.yml:6]: Correct. Just discussed in in slack with @praveenkumarct

**Comment by praveenkumarct** [comment]
Somehow the schedule was not working for the branch. So I tested directly in Master branch and it is working. So I will close this PR.

**Pull request closed without merging**